values.yaml

```diff
@@ -212,8 +212,13 @@ st2web:
       cpu: "50m"
     limits:
       memory: "100Mi"
-  # TODO: Add Ingress and/or LoadBalancer setting as a way to expose service to public (#6)
+  # TODO: Add Ingress setting as a way to expose service to public (#6).
+  # For now, use LoadBalancer or NodePort (default to LoadBalancer).
   # ingress:
+  service:
+    # NodePort | LoadBalancer
+    type: "LoadBalancer"
```

Comment:
Not 100% sure, but maybe `NodePort` or `ClusterIP` (default K8s) would be a better default?
With an additional option to set it to `LoadBalancer` or other type.